pcsx2/DEV9/ATA/Commands/ATA_CmdDMA.cpp

```diff
@@ -62,7 +62,11 @@ void ATA::DRQCmdDMADataFromHost()
 }
 void ATA::PostCmdDMADataFromHost()
 {
-	QueueWrite(currentWriteSectors, currentWrite, currentWriteLength);
+	WriteQueueEntry entry;
+	entry.data = currentWrite;
+	entry.length = currentWriteLength;
+	entry.sector = currentWriteSectors;
+	writeQueue.Enqueue(entry);
```

Comment:
entry.next not initialised

Reply:
Reiterating what was said in the discord just for reference;

entry is of type WriteQueueEntry (defined within class ATA), this pr removed the next field, so this seems like a false positive.

I noticed that SimpleQueue also defines a structure called WriteQueueEntry (defined within the class) which does have a field called next.

In theory this shouldn't cause issues (as they are defined within there own respective classes), but I've renamed one of them

Reply:
Zero initializing the whole struct made codacy happy